.gitpod.yml

```diff
@@ -0,0 +1,18 @@
+# List the start up tasks. Learn more https://www.gitpod.io/docs/config-start-tasks/
+tasks:
+  - name: yarn
```

Comment:
We leverage npm workspaces to orchestrate tasks across our monorepo, with each package (such as ts, go, py, etc.) having its own unique dependencies. I'm planning to propose a `.gitpod.yml` that encompasses all requirements for the entire repository, examples included. I’ll commit it directly to your PR before merging if that works for you. 😊

Reply:
Sure that works with me!😇